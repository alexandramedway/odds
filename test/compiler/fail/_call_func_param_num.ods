do call = (f, x) ->
  return !f(x)

do being_called = () -> return true

do call(being_called, true)
