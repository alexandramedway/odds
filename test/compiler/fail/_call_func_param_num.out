[31mSemantic error[0m:
  Function 'call' expected argument of type Func(Any => Bool) but was passed Func( => Bool) instead
